app/controllers/account/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Account
+  class UsersController < Account::AccountController
+    before_action :find_user, only: %i[show edit update destroy]
+
+    def show; end
+
+    def edit
+      @user = current_user
```

Comment:
you are reassigning the instance variable `@user`(`@user = User.find(params[:id])` from `before_action :find_user`)

idiomatically, that isn't a good approach, that is conflicting with REST: `get /users/5/edit` will allow you to edit the user with id 26(`current_user`) despite URL with id 5

delegate authorization function to pundit:
```
# user_policy.rb
def edit?
  user_admin? || current_user.id == model.id
end
```

Reply:
done